Source/Scene/Cesium3DTile.js

```diff
@@ -349,10 +327,19 @@ define([
 
         /**
          * Marks whether the tile's children bounds are fully contained within the tile's bounds
-         * 
+         *
          * @type {Cesium3DTileOptimizationHint}
          */
         this._optimChildrenWithinParent = Cesium3DTileOptimizationHint.NOT_COMPUTED;
+
+        this._sse = 0;
+        this._finalResolution = true;
+        this._loadHeap = undefined;
+        this._depth = 0;
+        this._centerZDepth = 0;
+        this._stackLength = 0;
+        this._selectedFrame = -1;
```

Comment:
_selectedFrame is used to decide what tiles to call `selectTile` on. Because we mark the tiles for selection separate from when we actually select them, checking `_selectedFrame` makes sure we don't end up selecting old stuff that was previously marked (because non-visible tiles don't get unmarked). `lastSelectedFrameNumber` looks like it's used to determine when to style tiles.